problems/dc_dc.jl

```diff
@@ -88,6 +88,8 @@ function system(;
             measnoise,
             nsys,
         )
+    elseif approx_mode == "δ-GAS Lyapunov"
+        contsys = ST.NewSimpleSystem(tstep, F_sys, measnoise, nsys)
```

Comment:
If none of the string match, you have an error later saying that `contsys` is not defined. At least with an `Enum`, we know we are in either of the three case :)